README.md

```diff
@@ -55,7 +55,7 @@ Environment variable and package.json configuration should be **strings**.
 Reporter options should also be strings exception for suiteNameTemplate, classNameTemplate, titleNameTemplate that can also accept a function returning a string.
 
 | Environment Variable Name | Reporter Config Name| Description | Default | Possible Injection Values
-|--|--|--|--|--|
+|---|---|---|---|---|
```

Comment:
Not strictly necessary, but broke my IntelliJ parser, and markup docs say it should be 3 dashes :-)